core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -562,10 +577,16 @@ $(document).ready(function () {
                         }
                     ],
                     "columnDefs": [
-                        { "visible": false, "targets": 15 },
-                        { "visible": false, "targets": 16 },
-                        { "visible": false, "targets": 17 },
-                        { "visible": false, "targets": 18 }
+                        // String with structures like : 'bytes / bytes', 'bytes / bytes'
+                        // they should be sorted as numerical-order instead of lexicographical-order.
+                        { "type": "size", "targets": 9 },
+                        { "type": "size", "targets": 10 },
+                        { "type": "size", "targets": 11 },
+                        { "type": "size", "targets": 12 },
+                        { "type": "size", "visible": false, "targets": 15 },
```

Comment:
Why `13` and `14` are not cared?

Reply:
I have modified the detailed description of the reason for the modification. 
I only modified the columns corresponding to 9-12 because the true values of these columns are actually strings, while columns 13-18 are all numbers indeed. Therefore, we only focused on columns 9-12 to sort them according to their numerical value.

Reply:
The type of col 15 - 18 seems to be number but how about col 13 and col14? They seems to be text.
https://github.com/apache/spark/blob/b17a0e6931cac98cc839c047b1b5d4ea6d052009/core/src/main/resources/org/apache/spark/ui/static/stagepage.js#L482

BTW, please don't resolve conversations. It's difficult to confirm what we pointed out is resolved.

Reply:
Hm,Infact col 13 and col 14 are actually numbers.
when it calles formbatBytes, the value of `type` variable equals `type`, so in function `formtBytes`, it will direcly returns the bytes instead of the converted value like `number + unit`.
```javascript
function formatBytes(bytes, type) {
    if (type !== 'display') return bytes;
    if (bytes <= 0) return '0.0 B';
    var k = 1024;
    var dm = 1;
    var sizes = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
    var i = Math.floor(Math.log(bytes) / Math.log(k));
    return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
}
```

Reply:
Oh, I see. `fromBytes` can return numbers.